libstuff/libstuff.h

```diff
@@ -145,7 +140,7 @@ struct SData {
 };
 
 // --------------------------------------------------------------------------
-// Time stuff
+// Time stuff TODO: Replace with std::chrono
```

Comment:
yep, in total support of that.  This might be a place where we can engage the community (or Upwork) to go through and do this kind of refactoring.